language/diem-vm/src/diem_transaction_executor.rs

```diff
@@ -490,7 +490,7 @@ impl DiemVM {
 
         let (round, timestamp, previous_vote, proposer) = block_metadata.clone().into_inner();
 
-        println!("========================================== {} ==========================================", round);
+        info!("========================================== {} ==========================================", round);
```

Comment:
The reason we didn't make this INFO, is because most people run with RUST_LOG=error because of all the noise. So it would suppress this log as well which most people want to see. Any other suggestions how to do it?

Reply:
Yeah, then might as well change this back.  Though, it would be nice to include the epoch info which isn't here.

Reply:
or we use error!(...) level to log